src/translations/en.json

```diff
@@ -57,7 +57,8 @@
       "save": "Save",
       "aNewVersionIsOut": "Version {{version}} of Archifiltre is out!",
       "downloadIt": "download it!",
-      "csvWithHash": "CSV with hashes"
+      "csvWithHash": "CSV with hashes",
+      "csvWithHashDisabledMessage": "You must wait for hashes to have been computed"
```

Comment:
You must wait for hashes to be computed

Reply:
Hum, je ne suis pas d'accord. Grammaticalement, to have been computed est bien plus correct.